lib/RateLimiterRedis.js

```diff
@@ -58,6 +61,28 @@ class RateLimiterRedis extends RateLimiterStoreAbstract {
     return new Promise((resolve, reject) => {
       const secDuration = Math.floor(msDuration / 1000);
       const multi = this.client.multi();
+
+      if (this.isClientV4) {
+        if (forceExpire) {
+          if (secDuration > 0) {
+            multi.set(rlKey, points, { EX: secDuration });
+          } else {
+            multi.set(rlKey, points);
+          }
+
+          return multi.pTTL(rlKey).exec(true);
+        }
+
+        if (secDuration > 0) {
+          return this.client.eval(this._scripts.increment, {
```

Comment:
This eval won't work, since `this._scripts.increment` does not point to anything (should use `incrTtlLuaScript`).